src/unity/python/turicreate/toolkits/_supervised_learning.py

```diff
@@ -295,7 +295,8 @@ def print_validation_track_notification():
 
 
 def create(dataset, target, model_name, features=None,
-           validation_set='auto', verbose=True, distributed='auto', **kwargs):
+           validation_set='auto', distributed='auto',
```

Comment:
I think this is potentially a breaking API change, if callers were using positional arguments for `verbose` (which is now `distributed`) or `distributed` (which is now `seed`). Can we keep the order the same as before and add `seed` at the end of the positional arguments?

Reply:
Done.